src/products/mappers/products.mapper.ts

```diff
@@ -0,0 +1,14 @@
+import { ProductDTO } from '@products/dtos/product.dto';
+import { ProductEntity } from '@products/entities/product.entity';
+
+export class ProductsMapper {
+  static entityToDTO(entity: ProductEntity) {
+    return new ProductDTO(
+      entity.name,
```

Comment:
Yes but I remove the DTO for now because I'm not currently requesting information. Just returning it using entities. But I will have this in mind for the next time